docker-compose.yml

```diff
@@ -29,7 +29,6 @@ x-cm-variables: &cm
     PALACE_TEST_MINIO_USER: "palace"
     PALACE_TEST_MINIO_PASSWORD: "test123456789"
     PALACE_TEST_REDIS_URL: "redis://redis:6379/3"
-    PALACE_TEST_GOOGLE_DRIVE_SERVICE_ACCOUNT_INFO_JSON: "${PALACE_TEST_GOOGLE_DRIVE_SERVICE_ACCOUNT_INFO_JSON}"
```

Comment:
If we're removing this here, it should probably get removed from all the CI stuff as well.